app/cli/lib/apply.go

```
@@ -55,176 +98,687 @@ func MustApplyPlan(planId, branch string, autoConfirm bool) {
 			if err != nil {
-				onErr("failed to read %s:", dstPath)
-				return
+				onErr("failed to get rollback confirmation user input: %s", err)
 			}
 
-			// Check if the file has changed
-			if string(bytes) == content {
-				// log.Println("File is unchanged, skipping")
-				continue
+			if res == string(types.ApplyRollbackOptionRollback) {
+				Rollback(toRollback, true)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)